backend/audit/views.py

```diff
@@ -439,6 +439,9 @@ def get(self, request, *args, **kwargs):
 
         try:
             sac = SingleAuditChecklist.objects.get(report_id=report_id)
+            audit_report = SingleAuditReportFile.objects.filter(sac_id=sac.id).latest(
```

Comment:
This will fail if there's no matching `SingleAuditReportFile`—which will be the case for any submissions where someone hasn't uploaded one yet. I think this probably needs its own try/catch block, although maybe there's a nicer way to do it.